internals/cli/cli.go

```diff
@@ -141,15 +141,15 @@ func fixupArg(optName string) string {
 }
 
 type clientSetter interface {
-	setClient(*client.Client)
+	setClient(client.ClientGetter)
 }
 
-type clientMixin struct {
-	client *client.Client
+type ClientMixin struct {
+	client.ClientGetter
 }
 
-func (ch *clientMixin) setClient(cli *client.Client) {
-	ch.client = cli
+func (ch *ClientMixin) setClient(cg client.ClientGetter) {
+	ch.ClientGetter = cg
```

Comment:
Do we need a GetClient() to get the address of the derivative Client based structure on the Pebble derivative CLI command side?